pkgs/development/tools/analysis/cpplint/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonApplication, fetchFromGitHub }:
```

Comment:
This is not a Python library, so pass in `python` or `pythonPackages` and use its attributes (`buildPythonApplication`).